cmd/kube-apiserver/app/options/options.go

```diff
@@ -117,6 +118,7 @@ func NewServerRunOptions() *ServerRunOptions {
 		EndpointReconcilerType:            string(reconcilers.LeaseEndpointReconcilerType),
 		IdentityLeaseDurationSeconds:      3600,
 		IdentityLeaseRenewIntervalSeconds: 10,
+		IdentityLeaseGCCheckPeriodSeconds: 3600,
```

Comment:
Does this need to be separate from IdentityLeaseDurationSeconds?

Reply:
good point. If we know all leases have the same duration (which is the case here), there is no point to configure the check period differently. Reverted

the knob made sense when/if:
1. the controller lived in KCM-- not true anymore
2. we want the controller to manage leases from aggregated servers (with different duration)-- the more I think about it, aggregated servers shouldn't rely on this GC controller: 
  a. they don't need heartbeat-based identity at all-- downward API and owner reference is more efficient
  b. if they really want, they can start a similar GC controller that operates their leases in a user namespace

Reply:
(just adding some thoughts why we don't need a separate flag) another benefit of having this flag is to make the GC more responsive-- the controller deletes a lease if the apiserver failed to refresh its lease in:
- `(leaseDuration, leaseDuration+checkPeriod]` seconds

instead of (without the flag)
- `(leaseDuration, 2*leaseDuration]` seconds

but that is not our goal. [We prefer that expired Leases remain for a longer duration as opposed to collecting them quickly](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/1965-kube-apiserver-identity/README.md#caveats). That's why we choice a super long leaseDuration by default (1h, compared with 10s renewInterval). If people want to GC leases sooner after an apiserver is not active, they should start with shortening the leaseDuration. Having a separate flag for this level of control seems like an overkill for an alpha feature.